services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -305,7 +304,7 @@ export const DocumentTable = ({
       .filter((row) => row.is_latest_version)
       .map((filteredRows) => filteredRows.document_manager_guid);
 
-    setProjectTitle(rowSelection[0].project_title);
+    setEntityTitle(rowSelection[0].project_title);
```

Comment:
Can we set the _document's_ entity_title to equal project title so that we're not specific to project files in this component? Like in the model itself, this.entity_title = jsonObject.project_title.

Reply:
The document model file has been adjusted for this now.